src/lm/arpa-file-parser-test.cc

```diff
@@ -160,9 +160,9 @@ void ReadIntegerLmLogconvExpectSuccess() {
 
   static std::string integer_lm = "\
 \\data\\\n\
-ngram 1=4\n\
-ngram 2=2\n\
-ngram 3=2\n\
+ngram 1 = 4\n\
```

Comment:
you should revert these changes.... the rules don't apply inside strings (would be surprised if cpplint.py noticed this).... and the test code should test the normal arpa format which the old string did.